Author: gregor herrmann <gregoa@debian.org>
Description: there seems to be a logical twist ...
Debian-specific: no

--- a/t/12_xs.t
+++ b/t/12_xs.t
@@ -13,9 +13,9 @@
 BEGIN {
 	eval { require Class::XSAccessor };
 	if ( ! $@ and Class::XSAccessor->VERSION and Class::XSAccessor->VERSION >= 1.05 ) {
-		plan( skip_all => 'Class::XSAccessor 1.05 is not installed' );
-	} else {
 		plan( tests => 7 );
+	} else {
+		plan( skip_all => 'Class::XSAccessor 1.05 is not installed' );
 	}
 }
 use File::Spec::Functions ':ALL';
--- a/t/15_array_xs.t
+++ b/t/15_array_xs.t
@@ -13,9 +13,9 @@
 BEGIN {
 	eval { require Class::XSAccessor::Array };
 	if ( ! $@ and Class::XSAccessor::Array->VERSION and Class::XSAccessor::Array->VERSION >= 1.05 ) {
-		plan( skip_all => 'Class::XSAccessor::Array 1.05 is not installed' );
-	} else {
 		plan( tests => 7 );
+	} else {
+		plan( skip_all => 'Class::XSAccessor::Array 1.05 is not installed' );
 	}
 }
 use File::Spec::Functions ':ALL';
